core/urls.py

```diff
@@ -21,6 +23,7 @@
         name="complete_action",
     ),
     path("checklist", views.checklist, name="checklist"),
+    path("<int:young_person_id>/checklist", views.checklist, name="yp_checklist"),
```

Comment:
I noticed that this path produces an error on my setup with a PA. I don't think you've done the checklist paths yet though, so isn't part of this PR?   This could be fixed by creating a slightly different URL:

```
path("<int:young_person_id>/checklist", views.yp_checklist, name="yp_checklist"),
```

Then, create the matching view which imports a young_person_id argument.